### PR TITLE
NAS-114284 / None / NAS-114284: Replace simple instances of == with ===

### PR DESCRIPTION
**Pull request by Raz-Dva**
No tests needed

### CODE REVIEW

File: src/app/modules/entity/entity-form/components/form-scheduler/form-scheduler.component.ts [hunk 1]
```diff
@@ -288,7 +288,7 @@ export class FormSchedulerComponent implements Field, OnInit, AfterViewInit, Aft
     this.control.valueChanges.pipe(untilDestroyed(this)).subscribe((evt: string) => {
       this.crontab = evt;
 
-      const isPreset: boolean = this.presets.filter((preset) => evt == preset.value).length != 0;
+      const isPreset: boolean = this.presets.filter((preset) => evt == preset.value).length !== 0;
```
Comment (dszidi) [inline src/app/modules/entity/entity-form/components/form-scheduler/form-scheduler.component.ts:291]: evt === preset.value

**Review by dszidi — COMMENTED** [review]
Just that one change is needed. The rest LGTM

File: src/app/modules/entity/entity-form/components/form-scheduler/form-scheduler.component.ts (hunk 1 above)
Comment (Raz-Dva) [reply src/app/modules/entity/entity-form/components/form-scheduler/form-scheduler.component.ts]: Fixed according to comment

**Review by dszidi — APPROVED** [review]
LGTM

**Pull request merged**